### DISCUSSION TITLE
Exclude specific items from specific scheduled backup

### DISCUSSION THREAD

**Discussion by Mounica-vl** [vmware-tanzu/velero#4097] [Community support Q&A] [answered]
I am trying to take backup of one entire namespace, but want to exclude certain PVC's from the namespace in the backup. Is there any specific command to achieve this scenario? Thanks in advance

**Comment by benjaminxie** [comment, marked as answer]
Are you using `velero create backup` ?

Looking at the help (using the `-h` flag), I think one of either `--snapshot-volumes=false` and `--exclude-resources pvc,pv` could work but I can't make any promises.

Good luck!

**Comment by Makovec** [comment]
Hi,
there is also additional option. If you look to `-h` there is `--selector` flag where you can specify resources with matching label.

David

**Comment by Makovec** [comment]
And - BTW - there exists this doc: https://velero.io/docs/v1.6/resource-filtering/

**Reply by Mounica-vl** [reply]
Hello @Makovec thanks for the response. can you please look into my latest  question in the discussion and provide some input. Thanks :D
